inc/notificationtargetcommonitilobject.class.php

```diff
@@ -55,6 +55,23 @@ function __construct($entity='', $event='', $object=null, $options=array()) {
       $this->options['sendprivate'] = true;
    }
 
+   /**
+    * Get events related to ItiObject
+   **/
+   function getEvents() {
+
+      $events = array('requester_user'    => __('Addition of a user in requesters'),
+                      'requester_group'   => __('Addition of a group in requesters'),
+                      'observer_user'     => __('Addition of a user in observers'),
+                      'observer_group'    => __('Addition of a group in observers'),
+                      'assign_user'       => __('Assignation to a technician'),
```

Comment:
No, we can't talk about assignment and requester in the same sentence, i think.
My problem is we don't have a common name for attribution like requester and observer.

My thought upper in this conversation is i found strange to use technician and supplier in the same sentence. Attribution is better i think.

Maybe we can stay for the initial proposal for requester and observer and use this sentence for the rest:
"Addition of a _type in attribution"
or
"Addition of a _type in assignees"